src/main/java/Binks.java

```diff
@@ -0,0 +1,81 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Binks {
+    /**
```

Comment:
Global variables in the execution process are minimised through the packing with methods. This greatly enhance readability.